### ISSUE TITLE
version 0.4.4 regression of issue #32

### ISSUE THREAD

**Issue by kguilbert** [py-bson/bson#52]
bson seems to have a regression from 0.4.3 to 0.4.4 which was originally reported in issue #32. Dumping/loading binary data doesn't work anymore. I'm able to reproduce this in python 3.4.3 (seen below) and python 3.5.1.

```
(venv343) kguilbert@kg-ubuntu:~$ pip install -q bson==0.4.3 && echo ok
ok
(venv343) kguilbert@kg-ubuntu:~$ python
Python 3.4.3 (default, Nov 17 2016, 01:08:31) 
[GCC 4.8.4] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import bson
>>> bson.loads(bson.dumps({'d':b'qwerty'}))
{'d': b'qwerty'}
>>> 
(venv343) kguilbert@kg-ubuntu:~$ pip install -q bson==0.4.4 && echo ok
ok
(venv343) kguilbert@kg-ubuntu:~$ python
Python 3.4.3 (default, Nov 17 2016, 01:08:31) 
[GCC 4.8.4] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import bson
>>> bson.loads(bson.dumps({'d':b'qwerty'}))
{'d': b''}
>>> 
```

**Comment by Parkayun** [comment]
Thanks for your report!

**Comment by Parkayun** [comment]
It fixed and released on 0.4.5!

**Comment by kguilbert** [comment]
Thanks! Has the new release been uploaded to pypi? I'm not able to update it with pip.
Also the listing http://pypi.python.org/simple/bson/ (where pip downloads from, i think) is missing 0.4.5.

**Comment by Parkayun** [comment]
@kguilbert I uploaded again with `bson__0.4.5.tar.gz`.

**Comment by kguilbert** [comment]
```
$ pip3 download bson==0.4.5
Collecting bson==0.4.5
  Could not find a version that satisfies the requirement bson==0.4.5 (from versions: 0.3.4, 0.3.5, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.4.4)
No matching distribution found for bson==0.4.5
```

Am I doing this wrong?

**Comment by Parkayun** [comment]
@kguilbert I think there some version issue with pypi. so I re-released with 0.4.6.
Have a try `pip download bson==0.4.6`

**Comment by kguilbert** [comment]
@Parkayun that did it, thanks!
